Helper/Config.php

```diff
@@ -666,7 +666,7 @@ public function getMediaImportImagesColumns()
      *
      * @return array
      */
-    public function getMediaImportGalleryColumns()
+    public function getMediaImportGalleryColumns($raw = false)
```

Comment:
`$raw = false` is for compatibility, if other modules may be using this helper as well.